src/org/opendatakit/briefcase/model/FormStatus.java

```diff
@@ -25,6 +25,7 @@
   private String statusString = "";
   private final StringBuilder statusHistory = new StringBuilder();
   private boolean isSuccessful = true;
+  private int historyMaxSize = 8096;
```

Comment:
I answered your comment below first and I'm having a little trouble reconciling it with this one! If updating the `JEditorPane` doesn't require allocating new objects, is there really a need for such a low limit? Is this because your machine is very memory constrained? I'm not saying the optimization is a bad one but I just want to make sure we have a record of why it's needed so we don't scratch our heads too much later.

I believe 8Kb is actually 8192 bytes which makes the choice of 8096 seem more meaningful than it is.